Services/CO.CDP.DataSharing.WebApi/Api/DataSharing.cs

```diff
@@ -1,188 +1,152 @@
-using System.ComponentModel.DataAnnotations;
-using Microsoft.OpenApi.Models;
+using System.Reflection;
+using CO.CDP.DataSharing.WebApi.Model;
 using DotSwashbuckle.AspNetCore.SwaggerGen;
+using Microsoft.OpenApi.Models;
 
 namespace CO.CDP.DataSharing.WebApi.Api
```

Comment:
file scoped namespaces for consistency, maybe fix forward?

Reply:
Done!